blog/convert-raw-trace-data/index.qmd

```diff
@@ -0,0 +1,223 @@
+---
+title: "Convert raw TRACE data to a local SQLite database"
+author: [Kevin Riehl, Lukas Müller]
+date: "2023-06-08"
+description: An R code that converts TRACE files from FINRA into a SQLite for facilitated analysis and filtering.
+image: thumbnail.jpg
```

Comment:
Thumbnail image is missing

Reply:
Uploaded file and changed accordingly,
picture is royalty free from here
https://unsplash.com/de/fotos/-VQMnroeFK4